test/process_wrapper/rustc_quit_on_rmeta.rs

```diff
@@ -0,0 +1,89 @@
+use std::path::PathBuf;
+use std::process::Command;
+use std::str;
+
+use runfiles::Runfiles;
+
+fn fake_rustc(process_wrapper_args: &[&'static str]) -> String {
+    let r = Runfiles::create().unwrap();
+    let fake_rustc = r.rlocation(
+        &[
+            "rules_rust",
+            "test",
+            "process_wrapper",
+            if cfg!(unix) {
+                "fake_rustc"
+            } else {
+                "fake_rustc.exe"
+            },
+        ]
+        .iter()
+        .collect::<PathBuf>(),
+    );
+
+    let process_wrapper = r.rlocation(
+        &[
+            "rules_rust",
+            "util",
+            "process_wrapper",
+            if cfg!(unix) {
+                "process_wrapper"
+            } else {
+                "process_wrapper.exe"
+            },
+        ]
+        .iter()
+        .collect::<PathBuf>(),
+    );
+
+    let output = Command::new(process_wrapper)
+        .args(process_wrapper_args)
+        .arg("--")
+        .arg(fake_rustc)
+        .output()
+        .unwrap();
+
+    assert!(
+        output.status.success(),
+        "unable to run process_wrapper: {} {}",
+        str::from_utf8(&output.stdout).unwrap(),
+        str::from_utf8(&output.stderr).unwrap(),
+    );
+
+    String::from_utf8(output.stderr).unwrap()
+}
+
+#[test]
```

Comment:
Can this be moved into a `tests` module?
```suggestion
#[cfg(test)]
mod test {
    #[test]
```